invoice/src/main/java/org/killbill/billing/invoice/api/user/DefaultInvoiceUserApi.java

```diff
@@ -482,4 +494,25 @@ private Invoice getInvoiceAndCheckCurrency(final UUID invoiceId, @Nullable final
         return invoice;
     }
 
+    @Override
+    public void invoiceStatusTransition(final UUID accountId, final UUID invoiceId, final CallContext context) throws InvoiceApiException {
+        final InternalCallContext internalCallContext = internalCallContextFactory.createInternalCallContext(accountId, context);
+        dao.changeInvoiceStatus(accountId, invoiceId, InvoiceStatus.COMMITTED, internalCallContext);
+        final Invoice invoice = this.getInvoice(invoiceId, context);
+
+        // notify invoice creation event
+        final DefaultInvoiceCreationEvent defaultInvoiceCreationEvent = new DefaultInvoiceCreationEvent(invoice.getId(), invoice.getAccountId(),
+                                                                                                        invoice.getBalance(), invoice.getCurrency(),
+                                                                                                        internalCallContext.getAccountRecordId(), internalCallContext.getTenantRecordId(), context.getUserToken());
+        postEvent(defaultInvoiceCreationEvent, accountId, internalCallContext);
```

Comment:
Why not? Intuitively, the behavior here should be the same as the one in `InvoiceDispatcher` when status is `COMMITTED`.